### DISCUSSION TITLE
Exporing submission data

### DISCUSSION THREAD

**Discussion by thunder87** [solspace/craft-freeform#358] [Questions & Answers] [answered]
Hi, I'm trying to export existing submission data through element-api, but the form-data returned is encrypted. Can't find any documentation on exporting the data. I have tried using both element-api and looking through the graphql endpoint.

**Comment by kjmartens** [comment, marked as answer]
Hi @thunder87, 

We haven't had anyone else ask about this before... would you be able to elaborate on this? And what do you mean by "encrypted" - I'm curious how the data looks. 🙂 

We also have plenty of built-in exporting options available within Freeform.
https://docs.solspace.com/craft/freeform/v3/overview/exporting.html
